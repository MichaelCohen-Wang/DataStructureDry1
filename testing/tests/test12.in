update_pirate_treasure 432 9
add_ship 368 7
get_treasure 234
update_pirate_treasure 234 0
add_pirate 328 368 20
get_richest_pirate 368
add_ship 165 15
add_pirate 94 368 30
get_cannons 48
add_ship 233 23
add_pirate 66 233 18
get_richest_pirate 37
add_ship 504 14
get_cannons 165
get_cannons 126
get_cannons 368
remove_pirate 66
get_treasure 328
ships_battle 83 319
add_pirate 376 358 16
add_ship 105 4
add_pirate 336 105 30
remove_ship 359
ships_battle 233 105
remove_pirate 328
update_pirate_treasure 94 7
treason 105 165
remove_ship 504
add_ship 185 28
ships_battle 165 185
add_ship 76 21
ships_battle 76 185