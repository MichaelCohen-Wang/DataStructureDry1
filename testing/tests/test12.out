update_pirate_treasure: FAILURE
add_ship: SUCCESS
get_treasure: FAILURE
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
get_richest_pirate: SUCCESS, 328
add_ship: SUCCESS
add_pirate: SUCCESS
get_cannons: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
get_richest_pirate: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 15
get_cannons: FAILURE
get_cannons: SUCCESS, 7
remove_pirate: SUCCESS
get_treasure: SUCCESS, 20
ships_battle: FAILURE
add_pirate: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
remove_ship: FAILURE
ships_battle: SUCCESS
remove_pirate: SUCCESS
update_pirate_treasure: SUCCESS
treason: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
